db/migrate/20241208071529_create_solid_queue_table.rb

```
@@ -0,0 +1,124 @@
+
+    create_table "solid_queue_semaphores", force: :cascade do |t|
+      t.string "key", null: false
+      t.integer "value", default: 1, null: false
+      t.datetime "expires_at", null: false
+      t.datetime "created_at", null: false
+      t.datetime "updated_at", null: false
+      t.index [ "expires_at" ], name: "index_solid_queue_semaphores_on_expires_at"
+      t.index [ "key", "value" ], name: "index_solid_queue_semaphores_on_key_and_value"
+      t.index [ "key" ], name: "index_solid_queue_semaphores_on_key", unique: true
+    end
+  end
```

rubocop (1.68.0)

[convention] Assignment Branch Condition size for change is too high. [<11, 99, 0> 99.61/15]

(Metrics/AbcSize)